jbmc/src/java_bytecode/ci_lazy_methods.cpp

```diff
@@ -210,6 +210,14 @@ bool ci_lazy_methodst::operator()(
     }
   }
 
+  for(const auto &class_name : instantiated_classes)
```

Comment:
It might be useful to document the 'why' here, as `cproverNondetInitialize` doesn't mean much for jbmc itself.

Reply:
`cproverNondetInitialize` is meaningful for jbmc, it allows to specify invariants of the data-structures used in the program that is analysed, (for instance the length of a list is always greater than 0)